src/deburr.ts

```diff
@@ -0,0 +1,82 @@
+/** Used to match Latin Unicode letters (excluding mathematical operators). */
+const reLatin = /[\xc0-\xd6\xd8-\xf6\xf8-\xff\u0100-\u017f]/g;
+
+/** Used to compose unicode character classes. */
+const rsComboMarksRange = '\\u0300-\\u036f\\ufe20-\\ufe23',
+	rsComboSymbolsRange = '\\u20d0-\\u20f0';
+
+/** Used to compose unicode capture groups. */
+const rsCombo = '[' + rsComboMarksRange + rsComboSymbolsRange + ']';
+
+/**
+ * Used to match [combining diacritical marks](https://en.wikipedia.org/wiki/Combining_Diacritical_Marks) and
+ * [combining diacritical marks for symbols](https://en.wikipedia.org/wiki/Combining_Diacritical_Marks_for_Symbols).
+ */
+const reComboMark = RegExp(rsCombo, 'g');
```

Comment:
Toto všetko presuňme nad funkcie kde sa to používa

Reply:
[fe789ee](https://github.com/xseman/bysquare/pull/32/commits/fe789ee5df8e98e4e82f7355e8e2b885e40c8516)